charts/internal/seed-controlplane/charts/hcloud-cloud-controller-manager/templates/cloud-controller-manager.yaml

```diff
@@ -69,6 +69,10 @@ spec:
           - name: HCLOUD_NETWORK
             value: {{ .Values.podNetworkIDs.workers | quote }}
           {{- end }}
+          {{- if .Values.disableIPv6 }}
+          - name: HCLOUD_LOAD_BALANCERS_DISABLE_IPV6
+            value: "true"
```

Comment:
Isn't that a bit redundant based on the value of `disableIPv6` being a boolean as well?